configure.ac

```diff
@@ -320,26 +320,30 @@ AS_IF([test "x$enable_proc" = "x"], [
 # check for ppp if not specified
 AC_PATH_PROG(PPP, [ppp], [/usr/sbin/ppp], "$PATH:/sbin:/usr/sbin")
 AS_IF([test "x$PPP_PATH" = "x"], [
-	AC_CHECK_FILE([$PPP], [
-		AS_IF([test "x$PPP_PATH" = "x"], [
-			PPP_PATH="$PPP"
-		])
-		AS_IF([test "x$with_ppp" = "x"], [
-			with_ppp="yes"
-		])
-	],[])
+	if test "$cross_compiling" != "yes"; then
+		AC_CHECK_FILE([$PPP], [
+			AS_IF([test "x$PPP_PATH" = "x"], [
+				PPP_PATH="$PPP"
+			])
+			AS_IF([test "x$with_ppp" = "x"], [
+				with_ppp="yes"
+			])
+		],[])
+	fi
```

Comment:
The idea was that you should explicitly specify the location of `pppd` when cross-compiling:
https://github.com/adrienverge/openfortivpn/blob/6b8f9fe/configure.ac#L274

But I get it, at least you don't get an obscure error message. Still, perhaps an error message is more appropriate, but we need a better one, complaining `if test "$cross_compiling" = "yes"` and `test "x$PPP_PATH" = "x"`.